specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-04-02-preview/managedClusters.json

```diff
@@ -4245,6 +4245,14 @@
         "disableRunCommand": {
           "type": "boolean",
           "description": "Whether to disable run command for the cluster or not."
+        },
+        "enableVnetIntegration": {
```

Comment:
Was there an internal review done for this? See [here](https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/232501/API-Designs-and-Proposals)

Reply:
yes:  https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki?wikiVersion=GBwikiMaster&pagePath=/AKS/Team%20Logistics/Overlay/Monthly%20API%20Review/2022%252D04%252D02%252Dpreview&pageId=232802&_a=edit

Reply:
For future reference we should also make a doc (at the link above), so that other stakeholders can discuss naming, etc. I think this change was envisioned before that process was set up though, so approving it